commit 48b5808fa6d1b373dee6921c2fecd310604d552e
Author: Jan Kobler <eng1@koblersystems.de>
Date:   Mon Nov 13 13:53:08 2017 +0100

    export symbol bitrate_table
    
    e.g. the symbol bitrate_table is used by cdrdao-1.2.3
    
    https://bugs.gentoo.org/635014
    
    Signed-off-by: Jan Kobler <eng1@koblersystems.de>

diff --git a/include/lame.def b/include/lame.def
index def5287..d73a642 100644
--- a/include/lame.def
+++ b/include/lame.def
@@ -253,6 +253,8 @@ lame_get_bitrate	@502
 lame_get_samplerate	@503
 lame_get_maximum_number_of_samples	@504
 
+bitrate_table @600
+
 lame_decode_init	@1000
 lame_decode		@1001
 lame_decode_headers	@1002
diff --git a/include/libmp3lame.sym b/include/libmp3lame.sym
index ff7d318..f51a883 100644
--- a/include/libmp3lame.sym
+++ b/include/libmp3lame.sym
@@ -232,3 +232,4 @@ id3tag_set_comment_utf16
 lame_get_bitrate
 lame_get_samplerate
 lame_get_maximum_number_of_samples
+bitrate_table

